motr/io_nw_xfer.c

```diff
@@ -1928,8 +1928,14 @@ static int nw_xfer_req_dispatch(struct nw_xfer_request *xfer)
 		 *    The units needed for recvoery are ready.
```

Comment:
Please fix the typo in the "recovery" word here.

Reply:
Done